### PR TITLE
Update HTTP -> HTTPS

### PR DESCRIPTION
**Pull request by him2him2**
URLs updated to use HTTPS protocol where appropriate to improve security and privacy.

### CODE REVIEW

**Comment by imba-tjd** [comment]
Well, I have opened a PR to do the same thing. https://github.com/tiangolo/fastapi/pull/2165

**Review by mickeypash — APPROVED** [review]
Lgtm

**Review by AdrianDeAnda — APPROVED** [review]

File: docs/en/docs/advanced/sql-databases-peewee.md [hunk 1]
```diff
@@ -376,7 +376,7 @@ Then run your app with Uvicorn:
 ```console
 $ uvicorn sql_app.main:app --reload
 
-<span style="color: green;">INFO</span>:     Uvicorn running on http://127.0.0.1:8000 (Press CTRL+C to quit)
+<span style="color: green;">INFO</span>:     Uvicorn running on https://127.0.0.1:8000 (Press CTRL+C to quit)
```
Comment (cahna) [inline docs/en/docs/advanced/sql-databases-peewee.md:379]: I don't think this line should not be changed.

**Review by ycd — COMMENTED** [review]
Why do we need https on localhost. In addition, If we revert them, we will have a PR that does exactly the same thing as #2165

File: docs/en/docs/advanced/sql-databases-peewee.md (hunk 1 above)
Comment (him2him2) [reply docs/en/docs/advanced/sql-databases-peewee.md]: Agreed. I have reverted  and committed the change. Thank you!

**Pull request closed without merging**